build.sbt

```diff
@@ -1,20 +1,14 @@
 import com.typesafe.sbt.packager.docker.Cmd
 
-val scala212 = "2.12.12"
-val scala211 = "2.11.12"
-val supportedScalaVersions = List(scala212, scala211)
-val sparkVersion: Option[(Long, Long)] => String = {
-  case Some((2, 11)) => "2.4.7"
-  case _ => "3.1.2"
-}
-val circeVersion: Option[(Long, Long)] => String = {
-  case Some((2, 11)) => "0.12.0-M3"
-  case _ => "0.14.0"
-}
+val scala212 = "2.12.15"
+val scala213 = "2.13.6"
```

Comment:
Can't bump to 2.13.7 since the build [can't solve the dependency on scalac-scoverage 1.4.8](https://github.com/cognitedata/cdp-spark-datasource/runs/4180435084?check_suite_focus=true#step:6:392).
Maybe we can force the dependency on [1.4.10 to support 2.13.7](https://repo1.maven.org/maven2/org/scoverage/scalac-scoverage-plugin_2.13.7/) later